setup.py

```diff
@@ -7,7 +7,7 @@
 
 setup(
     name = 'OpenFisca-Core',
-    version = '22.0.8',
+    version = '22.0.9',
```

Comment:
> The impact of the change is large enough that this could have warranted a minor bump IMO: it's easier to say my tax and benefit system relies on `v22.1+` rather than `v22.0.9+` 🙂